internal/api/web/views/pages/dashboard_pages/dashboard_home_page.templ

```diff
@@ -0,0 +1,38 @@
+package dashboard_pages
+
+import (
+	"github.com/Molnes/Nyhetsjeger/internal/api/web/views/components/layout_components"
+	"github.com/Molnes/Nyhetsjeger/internal/data/quizzes"
+	"github.com/Molnes/Nyhetsjeger/internal/api/web/views/components/dashboard_components/dashboard_home_page"
+)
+
+// Dashboard home page
+templ DashboardHomePage(unpublishedQuizzes []quizzes.Quiz, publishedQuizzes []quizzes.Quiz) {
+	@layout_components.DashBoardLayout() {
+		<div class="flex justify-center">
+			<div class="flex flex-col px-4 pt-8 max-w-screen-lg">
```

Comment:
Instead of having a wrapper only for centering the content, try using `margin-inline: auto` (`mx-auto` in Tailwind) instead on the element with the `max-w-screen` class, and remove the wrapper.

Reply:
good point, thanks  
fixed in 36719f8eb15053fd310dcf0f2bd82a4f3fcf5803